automatic/wixtoolset/wixtoolset.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>wixtoolset</id>
+    <version>3.11.0.1528-rc2</version>
+    <packageSourceUrl>https://github.com/Chocolatey/chocolatey-coreteampackages/tree/master/automatic/wixtoolset</packageSourceUrl>
+    <owners>chocolatey, Gary Ewan Park</owners>
+    <title>WiX Toolset</title>
+    <authors>Rob Mensching, Bob Arnson</authors>
+    <projectUrl>http://wixtoolset.org/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/bfbac575d697b67c7930af094980146278045677/icons/wixtoolset.png</iconUrl>
+    <copyright>2009 - 2016 Outercurve Foundation</copyright>
+    <licenseUrl>http://wixtoolset.org/about/license/</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/wixtoolset/wix3</projectSourceUrl>
+    <docsUrl>http://wixtoolset.org/documentation/</docsUrl>
+    <mailingListUrl>http://wixtoolset.org/documentation/</mailingListUrl>
+    <bugTrackerUrl>https://github.com/wixtoolset/issues/issues</bugTrackerUrl>
+    <tags>wixtoolset wix toolset msi xml admin</tags>
+    <summary>The most powerful set of tools available to create your Windows installation experience. Free and Open Source since 2004!</summary>
+    <description>
+The Windows Installer XML (WiX) is a toolset that builds Windows installation packages from XML source code. The toolset supports a command line environment that developers may integrate into their build processes to build MSI and MSM setup packages. The WiX toolset CodePlex project stores the stable releases and Mercurial repository.
```

Comment:
btw, the description is imported from the Readme.md file when the package gets updated.